src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString.split(';').reduce((prev, item) => {
```

Comment:
[CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead. The `reduce` method is used correctly here, which aligns with this guideline.